app/main.py

```diff
@@ -1,13 +1,11 @@
-# from models import Actor
-# from managers import ActorManager
-#
-# if __name__ == "__main__":
-#     Actor.objects = ActorManager()
-#
-#     Actor.objects.create(first_name="Emma", last_name="Watson")
-#     Actor.objects.create(first_name="Daniel", last_name="Radclife")
-#     print(Actor.objects.all())
-#     Actor.objects.update(2, "Daniel", "Radcliffe")
-#     print(Actor.objects.all())
-#     Actor.objects.delete(1)
-#     print(Actor.objects.all())
+from models import Actor
+from managers import ActorManager
+
+if __name__ == '__main__':
+    Actor.objects = ActorManager()
```

Comment:
The `Actor` class does not have an `objects` attribute. You need to ensure that the `Actor` class is designed to hold a reference to `ActorManager`, or alternatively, manage the `ActorManager` instance separately without attaching it to `Actor`.